src/qt/editor/welcomewidget.cpp

```diff
@@ -381,13 +323,7 @@ WelcomeWidget::WelcomeWidget(InviwoMainWindow* window, QWidget* parent)
 
     setTabOrder(filterLineEdit_, filetree_);
     setTabOrder(filetree_, loadWorkspaceBtn_);
-    setTabOrder(loadWorkspaceBtn_, newButton);
```

Comment:
did you check whether the tab order is still as expected?

Reply:
I just moved them, but I have not checked yet

Reply:
it still works, but we have no focus indication on the buttons...